README.md

```diff
@@ -160,9 +160,9 @@ Run migrations
 
 Build&run project
 
-    RUN_BOT=1 cargo run
-    RUN_WEB2=1 cargo run
-    RUN_WORKER=1 cargo run
+    cargo run bot
```

Comment:
These don't work, to pass args to the binary with cargo you have to put them after `--`, so `cargo run -- bot`